omniledger/ol/test.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 
-DBG_TEST=2
+DBG_TEST=1
 # Debug-level for app
-DBG_APP=2
-DBG_SRV=2
+DBG_APP=0
```

Comment:
I usually put `DBG_APP=1`, so you can at least see where the test failed in the jenkins output.